spec/house_spec.rb

```diff
@@ -0,0 +1,18 @@
+require './spec/spec_helper.rb'
+require_relative '../house.rb'
+# require 'byebug'
+
+describe House do
+    it 'returns the house that jack built rhyme' do
+        house = House.new('house.txt')
+        rhyme = house.rhyme
+
+        test_file = File.read('test_text.txt')
+
+        # puts "Test File:\n" + test_file
```

Comment:
Can you remove the comments `# puts ` unless you find them helpful for this project. You can always remove them in the end if you want to keep them until you finish the project.